desc/protoparse/validate.go

```diff
@@ -33,6 +37,25 @@ func validateBasic(res *parseResult, containsErrors bool) {
 	}
 }
 
+func validateImports(res *parseResult) error {
+	node := res.nodes[res.fd]
+	fileNode, _ := node.(*ast.FileNode)
```

Comment:
Why not `res.root?`?

```suggestion
	fileNode := res.root
	if fileNode == nil {
		return nil
	}
```

Reply:
Thanks